packages/react-native-nitro-modules/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "react-native-nitro-modules",
   "version": "0.14.0",
   "description": "Insanely fast native C++, Swift or Kotlin modules with a statically compiled binding layer to JSI.",
-  "main": "lib/index",
-  "module": "lib/index",
-  "types": "lib/index.d.ts",
```

Comment:
oh interesting - was this technically broken before?

Reply:
yes, the files couldn't be resolved before